endoreg_db/management/commands/import_video.py

```
@@ -6,6 +6,9 @@
 This command is designed to be run from the command line and takes various arguments
 to specify the video file, center name, and other options.
 """
+from curses import meta
```

Ruff (0.11.9)

`curses.meta` imported but unused

Remove unused import: `curses.meta`

(F401)